### DISCUSSION TITLE
Add ability to select multiple filter types at once in the World Hopper plugin

### DISCUSSION THREAD

**Discussion by Nick2bad4u** [runelite/runelite#16291] [Ideas]
![image](https://user-images.githubusercontent.com/20943337/219905363-c678d0a4-798b-4879-b014-de56329842f6.png)

**Comment by pwatts6060** [comment]
You can do this. Hold the Ctrl key and click multiple. You can also hold shift and click another entry and it selects all entries from the one selected to the next one you click. Similar to how selecting cells works in microsoft excel.
